CODE_OF_CONDUCT.md

```diff
@@ -1,46 +1,9 @@
-# Contributor Covenant Code of Conduct
+Welcome to the Weblate community
 
-## Our Pledge
+Accept and offer criticism constructively.
+Let anyone have the privacy they desire.
 
-In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
```

Comment:
This is not a pledge, as evident by its inclusion, and title. Further embodied in a seemingly active "we". It is redundant to law where it deals in harassment, and specifically, invokes a needless reiteration of how people are irrelevantly different to the task at hand. It is not plausible that any of this is needed, nor is it historically the case, bringing only into question its validity through tarnishing the project.